pipeline/build/azure-pipelines.yaml

```diff
@@ -15,14 +15,36 @@ resources:
 pool:
   vmImage: 'ubuntu-latest'
 
```

Comment:
need to short the triggers out, if this is merged into master, it will by default just run Dev

Reply:
rules are now as follows:
- PR (auto) will run against `dev` including destroy
- Merge to Master (auto) will run against `staging` not including destroy
- Manual allows selection of `dev` or `staging` (including manual override to not destroy dev)
* Nothing will allow staging to be deleted